tests/common_test.go

```diff
@@ -82,6 +85,12 @@ func LoadKey(in []byte) (priv crypto.Signer, err error) {
 	return derhelpers.ParsePrivateKeyDER(in)
 }
 
+// LoadURI attempts to load a private key from SOFTHSM.
+func LoadURI(uri string) (priv crypto.Signer, err error) {
+	pk11uri, _ := rfc7512.ParsePKCS11URI(uri)
+	return rfc7512.LoadPKCS11Key(pk11uri)
+}
```

Comment:
This exact function was already implemented somewhere else. Dedup?

Reply:
I think there was a decision made before defining `LoadKey` in `cmd/gokeyless/gokeyless.go` and passing it as a parameter, so I followed the same for `LoadURI`. We can discuss this and move them both to `server/` so they are importable.

Reply:
I made a change to fail if an error is returned and Github thought this is finished. It's not. I'll discuss it with Chris today.